### ISSUE TITLE
Status page not showing correct information

### ISSUE THREAD

**Issue by jhodges10** [dashpay/insight-ui#21]
Looks like the initial sync Start Date is broken.

It also appears that the Initial Blockchain Height and Synced Blocks are also broken, or not displaying the correct information for each data point.

![image](https://user-images.githubusercontent.com/18431344/41556426-92113378-72ef-11e8-8f8e-b5c33e13661f.png)

**Comment by nmarley** [comment]
I agree that "Start Date" is now broken and previously this wasn't the case. Not sure about the other fields, but for the record, a lot of this info is legacy and mostly a repeat of the data from `getinfo` call which is on the right sidebar.

Previously, Insight had a LevelDB k/v store which was used for address indexing, but that index has now been moved into the Core code, so it doesn't have to sync and index with DashCore anymore. (The "sync" fields now just display DashCore network sync info, as far as I understand it.)

**Comment by RaM1488** [comment]
)

**Comment by bfoss765** [comment]
@Marley Is the only open issue here the start date since the other items are available on the right sidebar via getinfo?

**Comment by bfoss765** [comment]
@Marley Is the only open issue here the start date since the other items are available on the right sidebar via getinfo?

**Comment by bfoss765** [comment]
Added to backlog.
